docs/guide1.0/app-cmsproperties.md

```diff
@@ -1,24 +1,22 @@
-CMS Page Properties
-========================
+# CMS Page Properties
 
-What are properties?
---------------------
+Page properties are personalised styles you want to apply to specific pages. 
 
-As the name already says, those are properties you can attach to a page you want. For example if you like to use different colors on different pages you can create a color property where the user can select a specific color for each page. Once the property is set you can use them in your view files for blocks or layouts.
+For example, if you want to use different colors on different pages, you can create a color property where the user can select a specific color for each page. Once the property is set you can use them in your view files for [Blocks](https://luya.io/guide/app-blocks) or [Layouts](https://luya.io/guide/app-cmslayouts) components.
```

Comment:
Thank you for the improvements.  To make links, just use the "internal" path to the markdown file, as they are all in the same folder just make:

`[Blocks](blocks.md)` instead. The luya.io website will automatically parse the links to its correct full resolved path.

With this approach the user can view the documentation on github very comfortable (https://github.com/luyadev/luya/blob/master/docs/guide1.0/app-blocks.md) or on luya.io.